ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/version/VersionClient.scala

```diff
@@ -16,4 +16,10 @@ final class VersionClient(ledgerId: LedgerId, service: VersionServiceStub) {
   )(implicit executionContext: ExecutionContext): Future[String] =
     it.getApiVersion(ledgerId, token)
 
+  def getApiFeatures(
+      ledgerIdToUse: LedgerId,
+      token: Option[String] = None,
+  )(implicit executionContext: ExecutionContext): Future[Seq[Feature]] =
```

Comment:
One thing I forgot: a `Set` is possibly a better type for API features.

Reply:
I don't really see the advantage of converting this into a `Set`. Users are also unique and non-ordered, but we return them in a `Seq`. Yes, the logic to check whether a feature is enabled is `O(n)` when using a `Seq`, but it seems unlikely `n` will ever get big enough to matter ;-)